### DISCUSSION TITLE
Some questions about tracing Redisson Lock

### DISCUSSION THREAD

**Discussion by peachisai** [open-telemetry/opentelemetry-java-instrumentation#10574] [Q&A] [answered]
Hello, I  want to contribute a Pr about tracing ```Redisson lock```。 
It will record lock name, lease time, wait time and thread name, I think it is a internal span, 
but the ```RedissonInstrumenterFactory``` is alway client span
It is ok to create another ```Instrumenter```？

**Comment by trask** [comment]
hi @peachisai, yes, it is ok for one instrumentation to have two Instrumenters

**Reply by peachisai** [reply, marked as answer]
Thanks for your reply
